docs/factoids/hardware-photos.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Using Get-Specs
```

Comment:
Think you forgot to change the title, lol

Reply:
Changed and fixed